com.vcvrack.Rack2.yaml

```diff
@@ -0,0 +1,76 @@
+id: com.vcvrack.Rack2
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '23.08'
+sdk: org.freedesktop.Sdk
+command: rack
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --allow=bluetooth
+  - --filesystem=host
+  - --filesystem=xdg-run/pipewire-0
+  - --env=XDG_CURRENT_DESKTOP=GNOME
+
+x-gl-version: &gl-version '1.4'
+x-gl-versions: &gl-versions 23.08;23.08-extra;1.4
+x-gl-merge-dirs: &gl-merge-dirs vulkan/icd.d;glvnd/egl_vendor.d;lib/dri;lib/d3d;vulkan/explicit_layer.d;vulkan/implicit_layer.d
+
+add-extensions:
+  org.freedesktop.Platform.GL:
```

Comment:
why is this necessary?

Reply:
It needed because of application is built with GLFW library - OpenGL is required for app to function properly.

Reply:
so you don't need it. The question was rhetorical.

Reply:
I've reworked manifest accordingly, double-checked, fixed.